libtcmu_common.h

```diff
@@ -146,13 +146,13 @@ void tcmu_unblock_device(struct tcmu_device *dev);
 void tcmu_flush_device(struct tcmu_device *dev);
 
 /* Helper routines for processing commands */
-char *tcmu_get_cfgfs_str(const char *path);
-int tcmu_set_cfgfs_str(const char *path, const char *val, int val_len);
-int tcmu_get_cfgfs_int(const char *path);
-int tcmu_set_cfgfs_ul(const char *path, unsigned long val);
-int tcmu_get_attribute(struct tcmu_device *dev, const char *name);
+char *tcmu_cfgfs_get_str(const char *path);
+int tcmu_cfgfs_set_str(const char *path, const char *val, int val_len);
+int tcmu_cfgfs_get_int(const char *path);
+int tcmu_cfgfs_set_ul(const char *path, unsigned long val);
+int tcmu_cfgfs_dev_get_attr(struct tcmu_device *dev, const char *name);
```

Comment:
May be you wanted to name it like tcmu_cfgfs_exec_dev_action()
i.e. tcmu_cfgfs_dev_get_attr() -> tcmu_cfgfs_get_dev_attr()